ui/app/rif/actions/index.js

```diff
@@ -707,8 +728,10 @@ function getTokensWithJoinedCheck () {
           const channels = Object.keys(channelObject).map(channelKey => channelObject[channelKey]);
           tokens.map(token => {
             const tokenJoined = token;
-            tokenJoined.joined = !!channels.find(channel => channel.token_address === ethUtils.toChecksumAddress(token.address));
             tokenJoined.openedChannels = channels.filter(channel => channel.token_address === ethUtils.toChecksumAddress(token.address));
+            tokenJoined.joined = !!channels.find(channel => channel.token_address === ethUtils.toChecksumAddress(token.address));
```

Comment:
I find this a little bit obscure to use the double negation, can we just evaluate the result of find? The result of find is undefined if it does not include the element, so this can be just

if(channels.find)

Reply:
Let me check if this solutions that you're suggesting works, later i'll comment it to you.